### ISSUE TITLE
Be more like prettyDiff?

### ISSUE THREAD

**Issue by tnrich** [royriojas/esformatter-jsx#52]
Here's how prettydiff is (at a high level) handling JSX: 
http://prettydiff.com/guide/react_jsx.xhtml

It seems to be the best JSX formatter that I've encountered thus far. Any chance you're doing something differently?

**Comment by royriojas** [comment]
hi @tnrich, interesting, I have been considering to switch to [jsbeautify](http://jsbeautifier.org/) directly since they now support jsx as well. 

But I would like to understand exactly what are the issues you're found using last version `esformatter-jsx`? If you can be specific it would be awesome. Will take a look a prettydiff as well.

**Comment by tnrich** [comment]
The main thing is esformatter-jsx doesn't seem to handle javascript
expression within jsx very well.

**Comment by royriojas** [comment]
Yes, I know. There are some improvements in the queue specifically for that. Do you have any specific code block that is failing? just curious?
